templates/base.html

```diff
@@ -0,0 +1,31 @@
+{% load static %}
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% block title %}
+        <title>Home</title>
```

Comment:
"Create block title with Taxi Service title inside"

Reply:
Title works. Its get title from context

Reply:
Yea i see, but for "normal" title must be not "Home", but Taxi Service. And if you want for page something else you change it
Now you must change block title every time, because you have name "Home"
And in task you create block title and Taxi Service inside it. Not "Change block title in ***.html"

Reply:
Okay, i have better example
If you have 10 page and want all of them have name just common "Taxi Service" you must write it in all 10 .html
So for base .html better name Taxi Service which have connect to site topic, not "Home"

Reply:
Well, now we start to think hypothetically what could be. If I had 10 such taxi pages, I would have something like base_taxi_home.html or a certain include.

Reply:
Okey, let's Polina/Alexei will make decision

Reply:
in this project, it's better to put the title 'Taxi service' in base.html in `{% block title %}`. The less code the better. Many pages will have this title. Also, when you start creating class-based views it will be a lot of work to pass title to context. But now it's not so obvious so it's okay to have one solution and then change it when you see a better variant

Reply:
in this project, it's better to put the title 'Taxi service' in base.html in {% block title %}. The less code the better. Many pages will have this title. Also, when you start creating class-based views it will be a lot of work to pass title to context. But now it's not so obvious so it's okay to have one solution and then change it when you see a better variant